src/component/PrintPanel/PrintPanelV3.tsx

```diff
@@ -645,7 +648,7 @@ export class PrintPanelV3 extends React.Component<PrintPanelV3Props, PrintPanelV
           </Col>
           {/* settings column */}
           <Col
-            span={12}
+            flex={2}
```

Comment:
cant we keep the old 50 / 50 split (setting this flex also to 1)?

Reply:
right, fixed.